sidebars.js

```diff
@@ -272,8 +601,608 @@ const sidebars = {
+          label: 'Release Notes',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-support-policy',
+          label: 'Release Support Policy',
+        },
+      ],
+    },
+  ],
+
+  // Japanese sidebar nav
+  docsJapanese: [
+    {
+      type: 'doc',
+      label: 'ScalarDL ドキュメンテーションホーム',
+      id: 'index',
+    },
+    {
+      'type': 'category',
+      'label': 'ScalarDL について',
+      'collapsible': true,
+      'items': [
+        {
+          type: 'doc',
+          id: 'overview',
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'implementation',
+          label: '実装',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'roadmap',
+          label: 'ロードマップ',
+        },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'はじめよう',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'getting-started',
+          label: 'ScalarDL Ledger を通じてコントラクトを実行する',
+        },
+        {
+          type: 'doc',
+          id: 'getting-started-auditor',
+          label: 'ScalarDL Ledger と ScalarDL Auditor を通じてコントラクトを実行する',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'サンプル',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'applications/simple-bank-account/README',
+          label: '銀行口座アプリケーション',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: '開発',
+      collapsible: true,
+      items: [
+        {
+          type: 'category',
+          label: 'SDKs',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardl-java-client-sdk/README',
+              label: 'ScalarDL Java クライアント SDK',
+            },
+          ],
+        },
+        {
+          type: 'doc',
+          id: 'configurations',
+          label: 'ScalarDL の設定',
+        },
+        {
+          type: 'doc',
+          id: 'schema-loader',
+          label: 'ScalarDL Schema Loader',
+        },
+        {
+          type: 'doc',
+          id: 'ca/caclient-getting-started',
+          label: '証明書を取得する',
+        },
+        {
+          type: 'doc',
+          id: 'authentication',
+          label: '認証を使用する',
+        },
+        {
+          type: 'doc',
+          id: 'how-to-handle-errors',
+          label: 'エラーを処理する',
+        },
+        {
+          type: 'doc',
+          id: 'how-to-write-contract',
+          label: 'コントラクトを作成する',
+        },
+        {
+          type: 'doc',
+          id: 'how-to-write-function',
+          label: 'ファンクションを書く',
+        },
+        {
+          type: 'doc',
+          id: 'use-generic-contracts',
+          label: '汎用コントラクトおよびファンクションの使用',
```

Comment:
Probably, this is consistent with others, and a short one might be better.

```suggestion
          label: '汎用コントラクトを使用する',
```

Reply:
Ah, thank you for mentioning this!

In a PR for the ScalarDB sidebar, [@/feeblefakie mentioned noun form and verb form](https://github.com/scalar-labs/docs-scalardb/pull/762/files/1c1b5404f5b23485105ae6a17a42977ecdc5aac3#r1869009005). In that PR, I changed them to noun form, so I've done the same in the sidebars in this PR as well in https://github.com/scalar-labs/docs-scalardl/pull/597/commits/7e4b2e058e63f387f7222548a350c7d8db97ba74.

Reply:
@josh-wong Thank you! Got it about the noun form thing.